src/scripts/ui/BuildingPage.tsx

```diff
@@ -83,9 +90,22 @@ export function BuildingPage(props: { tile: ITileData }): React.ReactNode {
    const gs = useGameState();
    const definition = Config.Building[building.type];
    const Body = BuildingBodyOverride[building.type] ?? DefaultBuildingBody;
+   const buildingByType = getTypeBuildings(gs);
+   const buildingCount = sizeOf(buildingByType.get(building.type));
+   let showBuildingCount = true;
+   if (
+      isHeadquarter(building.type) ||
```

Comment:
I remember `isSpecialBuilding` should include HQ and wonder, so it should be enough on its own?

Reply:
Ah so it does.  Removed the other 2 from the conditions.